package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@google/maps": "0.4.3",
     "body-parser": "1.17.2",
+    "cors": "^2.8.4",
```

Comment:
@kurtishouser please remove the `^` from this package to lock the dependency in place, thanks!